relevanceai/operations_new/cluster/models/sentence_transformers/community_detection.py

```diff
@@ -0,0 +1,132 @@
+"""Port over the community detection algorithm
```

Comment:
Communtiy detection implemented here is our own version to run on CPU/GPU if needed. Should it still be stored in `sentence_transformers`

Reply:
How about `models/local`?

Reply:
sounds good